soql-analyzer/src/test/scala/com/socrata/soql/SoQLAnalyzerTest.scala

```diff
@@ -501,4 +501,21 @@ SELECT visits, @x2.zx
       ColumnName("zx") -> ColumnRef(Some("_x2"), ColumnName("zx"), TestText)(NoPosition)
     ))
   }
+
+  test("window function") {
+    val analysisWordStyle = analyzer.analyzeUnchainedQuery("SELECT avg(visits) OVER (PARTITION BY name_last, 2, 3)")
+    val select = analysisWordStyle.selection.toSeq
+    select must equal (Seq(
+      ColumnName("avg_visits_name_last_2_3") -> typedExpression("avg(visits, name_last, 2, 3)")
```

Comment:
This is _really_ weird to me.  Can you explain why you chose this analysis result?

Reply:
In particular, why not have this compile into something like `op$over(avg(visits), name_last, 2, 3)` and then have the `op$over` "function" be a form treated specially by the sql interpreter?  As it is, it looks like it'd be easy for users to accidentally create a window call just by incorrectly passing multiple parameters to `avg`.

Reply:
I am not attached to the specific result.  op$over does look less weird.